src/templates/mediaproject-vite/src/store/models/AppStateModel/AppStateModel.ts

```diff
@@ -0,0 +1,56 @@
+import { action, computed, makeObservable, observable } from 'mobx';
+
+import { AppStateEnum } from './types';
+
+export default class AppStateModel {
+  state: AppStateEnum = AppStateEnum.notLoaded;
+
+  constructor() {
+    makeObservable(this, {
+      state: observable,
+
+      notLoaded: computed,
+      loading: computed,
+      loadedSuccessfully: computed,
+      loadedWithError: computed,
+
+      setState: action,
```

Comment:
```suggestion
      setState: action.bound,
```
Блин надо было сначала все поправить, а потом выпиливать шаблон...